app/models/current_claim.rb

```diff
@@ -144,6 +144,12 @@ def policy_year
     end
   end
 
+  def eligible_eligibility
```

Comment:
Wondering if this could just be `def eligibility`?

Reply:
@vassyz do you have to maintain an interface or something? Is that why it's got this name?

Reply:
No, it's literally an eligible eligibility (if there is one).